frontend/src/scenes/cohorts/cohortEditLogic.ts

```diff
@@ -195,14 +198,48 @@ export const cohortEditLogic = kea<cohortEditLogicType>([
                 setQuery: (state, { query }) => (isDataTableNode(query) ? query : state),
             },
         ],
+        creationPersonQuery: [
+            {
+                kind: NodeKind.DataTableNode,
+                source: {
+                    kind: NodeKind.ActorsQuery,
+                    fixedProperties: [],
+                    select: ['id', 'person_display_name -- Person'],
+                },
+                showPropertyFilter: false,
+                showEventFilter: false,
+                showExport: false,
+                showSearch: true,
+                showActions: false,
+                showElapsedTime: false,
+                showTimings: false,
+            } as DataTableNode,
+            {
+                setCreationPersonQuery: (state, { query }) => (isDataTableNode(query) ? query : state),
+            },
+        ],
+        personsToCreateStaticCohort: [
+            {} as Record<string, boolean>,
+            {
+                addPersonToCreateStaticCohort: (state, { personId }) => ({
+                    ...state,
+                    [personId]: true,
+                }),
+                removePersonFromCreateStaticCohort: (state, { personId }) => {
+                    const newState = { ...state }
+                    delete newState[personId]
+                    return newState
+                },
+            },
+        ],
     })),
 
-    forms(({ actions }) => ({
+    forms(({ actions, values }) => ({
         cohort: {
             defaults: NEW_COHORT,
-            errors: ({ id, name, csv, is_static, filters }) => ({
+            errors: ({ name, is_static, filters }) => ({
                 name: !name ? 'Cohort name cannot be empty' : undefined,
-                csv: is_static && id === 'new' && !csv ? 'You need to upload a CSV file' : (null as any),
+                csv: undefined,
```

Comment:
I removed the error check here because we want the user to be able to either upload a CSV file or select a user manually. 

I can't think of a way to use the form to collect the person ids, so the person ids are saved in a reducer logic. 

I moved the validation to the submit function instead.

Not ideal; appreciate any suggestions/feedback 🙏